crates/triangulation/src/face_triangulation.rs

```diff
@@ -694,6 +694,116 @@ pub fn sweeping_line_triangulation(edges: Vec<Segment>) -> (Vec<Triangle>, Vec<P
     )
 }
 
+/// Check if a polygon is convex.
+///
+/// This function determines if a given polygon is convex by examining its vertices.
+/// A polygon is convex if all its interior angles are less than or equal to 180 degrees
+/// and all vertices have the same orientation.
```

Comment:
The possible orientations include collinear, clockwise and couterclockwise?

Reply:
improved docstring